src/dashboards.js

```diff
@@ -323,4 +323,41 @@ var indicate = (function() {
       $(".busy-indicator").hide();
     }
   }
-})();
\ No newline at end of file
+})();
+
+function compressOSs() {
```

Comment:
hmm, so there might be a simpler way to do this.. in the OS box, have the top 3 options be, "Windows (all)", "Linux (all)", "OS X (all)".. and then some kind of divider followed by the actual os/version combos as additional options. And if a person manual selects all windows versions but doesn't check the "windows" box, then don't bother trying to "compress" it into os=WINNT.. it will be super-rare for someone to hurt themselves that way :)
Also, selecting a specific e.g. Windows version should clear the "All Windows" option (but not the All Linux or All OS X options"